DataFormats/FTLDigi/src/classes_def.xml

```diff
@@ -7,7 +7,9 @@
    <class name="edm::Wrapper<FTLDigiCollection>" />
    <class name="BTLSample"/>
    <class name="std::vector<BTLSample>"/>
-   <class name="ETLSample"/>
+   <class name="ETLSample" ClassVersion="10">
+     <version ClassVersion="10" checksum="1290724074"/>
```

Comment:
@parbol according to my understanding of the instructions linked in the twiki https://github.com/cms-sw/cmssw/pull/28349/files#r343383848 we should have a class version corresponding to the original ```ETLSample``` and then another one, with a differet checksum, for the updated version, with the additional member